config/schema.yaml

```diff
@@ -0,0 +1,327 @@
+# Precedence order (https://www.npmjs.com/package/convict)
+# When merging configuration values from different sources, Convict follows precedence rules. The order, from lowest to highest, for config.loadFile(file) and config.load(json) is:
+
+# Default value
+# File or json set in function argument
+# Environment variables (only used when env property is set in schema)
+# Command line arguments (only used when arg property is set in schema)
+
+# for easy json/yaml conversion: https://www.json2yaml.com/
+
+# NOTE: Please be careful when chosing defaults for undefined 
+#       environment variables. "undefined" is recommended
+
+# @TODO: update all formats (https://www.npmjs.com/package/convict)
+
+# The defaults for all variables below are currently for development builds.
+
+env:
+  doc: The application environment
+  format:
+  - production
+  - development
+  - test
+  default: development
+  env: NODE_ENV
+
+# for server
+aws_access_key_id:
+  tag: api
+  doc: aws_access_key_id
+  default: undefined
+  env: AWS_ACCESS_KEY_ID
+  sensitive: true
+aws_secret_access_key:
+  doc: aws_secret_access_key
+  default: undefined
+  env: AWS_SECRET_ACCESS_KEY
+  sensitive: true
+aws_region:
+  doc: aws_region
+  default: us-east-1
+  env: AWS_REGION
+dev_mode:
+  doc: dev_mode
+  default: true
+  env: DEV_MODE
+aws_s3_api_version:
+  doc: aws_s3_api_version
+  default: 2006-03-01
+  env: AWS_S3_API_VERSION
+stripe_client_id:
+  doc: stripe_client_id
+  default: undefined
+  env: STRIPE_CLIENT_ID
+stripe_secret_key:
+  doc: stripe_secret_key
+  default: sk_test_NFBDEThkpHCYBzXPJuBlY8TW
+  env: STRIPE_SECRET_KEY
+database_url:
+  doc: database_url 
+  default: postgres://postgres:oiPorg3Nrz0yqDLE@postgres:5432/polis-dev
+  env: DATABASE_URL
+database_for_reads_url:
+  doc: database_for_reads_url (often is the same as database_url)
+  default: postgres://postgres:oiPorg3Nrz0yqDLE@postgres:5432/polis-dev
+  env: DATABASE_FOR_READS_URL
+slack_api_token:
+  doc: slack_api_token
+  default: undefined
+  env: SLACK_API_TOKEN
+polis_slack_app_client_id:
+  doc: polis_slack_app_client_id
+  default: X
+  env: POLIS_SLACK_APP_CLIENT_ID
+polis_slack_app_client_secret:
+  doc: polis_slack_app_client_secret
+  default: X
+  env: POLIS_SLACK_APP_CLIENT_SECRET
+disable_plans:
+  doc: disable_plans
+  default: true
+  env: DISABLE_PLANS
+disable_intercom:
+  doc: disable_intercom
+  default: true
+  env: DISABLE_INTERCOM
+intercom_access_token:
+  doc: intercom_access_token
+  default: undefined
+  env: INTERCOM_ACCESS_TOKEN
+should_use_translation_api:
+  doc: should_use_translation_api
+  default: false
+  env: SHOULD_USE_TRANSLATION_API
+google_api_key:
+  doc: google_api_key
+  default: undefined
+  env: GOOGLE_API_KEY
+
+google_creds_stringified:
+  doc: google_creds_stringified
+  default: X
```

Comment:
Can you explain why "X" instead of empty string (assuming a default is required, right?)